www/locales/en.json

```diff
@@ -247,7 +247,7 @@
     },
     "WRITERS": {
       "ALL_WRITERS": {
-        "TEXT": "All Writers"
+        "TEXT": "All writers"
```

Comment:
i feel like the second word being capitalized improved readability

Reply:
@tarunsingh5 veerji, yeah I reverted this change back, this was a sideeffect of another text change we did earlier but wasn't pushed.